server/src/main/java/org/elasticsearch/rest/action/admin/indices/RestGetIndexTemplateAction.java

```diff
@@ -47,7 +49,11 @@
 
     private static final Set<String> RESPONSE_PARAMETERS = Collections.unmodifiableSet(Sets.union(
         Collections.singleton(INCLUDE_TYPE_NAME_PARAMETER), Settings.FORMAT_PARAMS));
-
+    private static final DeprecationLogger deprecationLogger = new DeprecationLogger(
+            LogManager.getLogger(RestGetIndexTemplateAction.class));
+    public static final String TYPES_DEPRECATION_MESSAGE = "[types removal] The response format of get index template requests will change"
+            + " in the next major version. Please start using the `include_type_name` parameter set to `false` in the request to "
```

Comment:
It's probably not worth another full build, but just in case you need to rebuild for another reason -- we decided not to use backticks in the messages.